.travis.yml

```diff
@@ -19,6 +19,11 @@ script:
   - docker-compose up --build --force-recreate -d
   - sleep 20
   - wget --no-check-certificate https://eid.local
+  - docker-compose exec api python manage.py creatersakey
+  - docker-compose exec api python manage.py createuser USER CLIENT_ID
```

Comment:
I believe it should read `$USER` instead of `USER`, as `$USER` is also added to the environment in `init_ci.sh` (same for the other 'variables')